src/content/blog/why-speak-at-jsheroes.md

```diff
@@ -0,0 +1,39 @@
+---
+author: alex-moldovan
+title: Why speak at JSHeroes
+published: Nov 8, 2024
+tags: ["cfp"]
+summary: A few reasons to convince you to apply as a speaker
+---
+
+In 2025 we’ll be running JSHeroes for the 7th time. In all our past editions we had amazing speakers on our stage. Maybe some of you reading this are already convinced and you want to be a speaker at our event, but for everyone else, let me try to articulate the "why".
```

Comment:
```suggestion
In 2025 we’ll be running JSHeroes for the 7th time. In all our past editions we had amazing speakers on stage. Some of you reading this might be already convinced to be a speaker at our event, but for everyone else, let me try to articulate the "why".
```